app/models/scan.rb

```diff
@@ -1,5 +1,4 @@
 class Scan < ActiveRecord::Base
-  belongs_to :event
```

Comment:
Ok so scans no longer belong to an event but do we still need the event_id attribute on the Scan model?